### DISCUSSION TITLE
Number of Accounts under a single Platform App

### DISCUSSION THREAD

**Discussion by mark47jacobs** [chatwoot/chatwoot#4821] [Q&A]
@pranavrajs From API docs, I understand that there is no limitation on the number of accounts that can be created under a single platform app. So when using the [account creation API](https://www.chatwoot.com/developers/api/#operation/create-an-account), should an upper limit be enforced for the number of accounts under a single platform app for optimal cache and DB performance? If that is the case is an API available for the creation of new platform apps, apart from the UI available in the super admin console. Thanks in advance.

**Comment by pranavrajs** [comment]
@mark47jacobs Can you please share the use-case/concern for limiting the number of accounts?

**Comment by mark47jacobs** [comment]
@pranavrajs we essentially do not want to limit the number of accounts under a platform app. Still, if creating many accounts under the same platform app would result in poor cache/DB performance we would want to limit the number of accounts. Either way, keeping a single platform app is easier for implementation on our end as well, so I just want to know your thoughts on that.

**Reply by sojan-official** [reply]
This should be fine. Most of our community members work with a single platform app. The distinction is primarily for permission purposes in rare cases where multiple apps might need to work over a single installation. But from a performance standpoint, there shouldn't be any worries.

**Comment by pranavrajs** [comment]
@mark47jacobs I'm not sure if I understand. Is there an issue with performance if there are a large (please specify the number) number of accounts under a single platform app on your installation? We have not seen/experienced such an issue yet.

**Reply by pranavrajs** [reply]
Moving this to a discussion as it doesn't seem to be a bug/issue.
